packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -43,8 +44,14 @@ export default function BlockNavigationList( {
 	blocks,
 	selectedBlockClientId,
 	selectBlock,
+	showAppender,
+
+	// Internal use only.
```

Comment:
This is clever and made me think about how I'd love to see props grouped into `// Public` and `//Private` for all our components. I'm always finding it tough to figure out whether a prop is public or e.g. added via `withSelect`.

(Just a thought—nothing to do with this PR!)